src/app/dto/pet.dto.go

```diff
@@ -28,6 +28,7 @@ type PetResponse struct {
 	Contact      string          `json:"contact"`
 	AdoptBy      string          `json:"adopt_by"`
 	Images       []ImageResponse `json:"images"`
+	IsLike *bool `json:"is_like"`
```

Comment:
Do we still have that bool problem? also, can you make this line align with the above?

Reply:
actually no if frontend send update form normally, i will change it to `bool`